rfc/rfc-0000-lightweight-rfc-process.md

```diff
@@ -0,0 +1,67 @@
+# RFC-0000: Lightweight RFC Process for riff
+
+**Authors:** Swapnil Bawaskar
+
+**Status:** **Active** | Accepted | Rejected | Superseded
+
+**Pull Request URL:** https://github.com/projectriff/riff/pull/1358
+
+**Superseded by:** N/A
+
+**Supersedes:** N/A
+
+**Related:** N/A
+
+
+## Problem
+As the scope of the project expands and the number of moving pieces increases, we need a way for contributors to propose designs for significant changes, and encourage discussion and feedback in a PR forum before proceeding with implementation.
+
+This document will refer to that process as the “Request For Comments (RFC) process”.
+
+## Solution
+The proposed solution to address the problem described above is to have an individual author or a group of authors submit a proposal to the community in the form of a RFC PR with a RFC markdown file, both called rfc-xxxx-<some-name>.md, in order to gather feedback and achieve consensus. The RFC follows the same format as used by this proposal.
+
+Much inspiration for this proposal has been drawn from [Apache Geode's RFC process](https://cwiki.apache.org/confluence/display/GEODE/Lightweight+RFC+Process), which in turn draws from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html).
+
+All RFCs are submitted via PRs to the [github.com/projectriff/riff](https://github.com/projectriff/riff) repo which will be merged when consensus has been reached about the state ("Accepted/Rejected").
+
+### Collaboration
+Comments and feedback should be provided on the PR.
+
+Authors should address all comments. This doesn't mean every comment and suggestion must be accepted and incorporated, but they should be carefully read and responded to.
+
+Every RFC is in one of the following phases:
+* **Active**: The RFC has been drafted and is open for comments. When making the PR, the doc should start out in this state.
```

Comment:
maybe "Pending" is more accurate for what this represents? (something that will transition to either Accepted or Rejected)

Reply:
or maybe "Proposed" and that will be discussed and either "Accepted" or "Rejected"

Reply:
Isn't Active/Pending implicit in that there is a PR open and Accepted implicit in that the PR was merged without being rejected?

Reply:
That's a good point. It feels odd that something in the doc itself has to be changed upon merging, but that's true regardless of whether the pre-merge status is "Active/Pending" or _blank_.

Reply:
It is meant to say that my "request for comments" is Active. I could be convince to remove "Active", but I think it is better to have an explicit "Accepted/Rejected" for new people.

Reply:
removed "Active" status.